worlds/ff4fe/FreeEnterpriseForAP/FreeEnt/f4c/lark/LICENSE

```diff
@@ -0,0 +1,18 @@
+Copyright © 2017 Erez Shinan
```

Comment:
lark is in here twice -- here and at `worlds/ff4fe/FreeEnterpriseForAP/f4c/lark/LICENSE`
It should probably only be in here once

Reply:
Turns out I didn't need the duplicate f4c module. anymore with me changing how the harp songs and z sprites work.